docs/sdks/server-sdks/node/quickstart.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: Quickstart
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Eppo Node.js SDK enables feature flags and experiments in your JavaScript applications with only a few lines of code.
+
+The SDK handles all the complexity of feature flag evaluation and experiment assignment locally in your application, with no network calls required after initial setup. This guide will walk you through installing the SDK and implementing your first feature flag, experiment, and contextual bandit.
+
+## Installation
+
+First, install the SDK into your current project.
+
+The Eppo Node SDK is available via a number of package managers.
+
+<Tabs>
+<TabItem value="yarn" label="Yarn">
+
+```bash
+yarn add @eppo/js-server-sdk
+```
+
+</TabItem>
+
+<TabItem value="npm" label="NPM">
+
+```bash
+npm install @eppo/js-server-sdk
+```
+
+</TabItem>
+
+<TabItem value="bun" label="Bun">
+
+```bash
+bun add @eppo/js-server-sdk
+```
+
+</TabItem>
+
+</Tabs>
+
+
+Depending on whether you are running a feature flag, an experiment, or a bandit, the initialization code will be slightly different.
+
+## Feature Flags
+Feature flags are a way to toggle features on and off without needing to deploy code.
+
+### Initialize the SDK
+
+[Create an SDK key](/sdks/sdk-keys) if you don't already have one. 
+
+First, initialize the SDK using your SDK key. The SDK key is different from the project API key. 
+
+
+```js
+import init from "@eppo/js-server-sdk";
+
+await init({apiKey: 'SDK_KEY'});
```

Comment:
Same as our JavaScript SDK, `assignmentLogger` is, unfortunately, required by TypeScript:
```ts
await init({
  apiKey: 'UNUSED',
  assignmentLogger: { logAssignment: (assignmentEvent)=> console.log('Send to warehouse: ', assignmentEvent) }
});
```